### ISSUE TITLE
JetEngine - Download media files of a CPT/CCT from the wordpress administration panel

### ISSUE THREAD

**Issue by viabit-dev** [CrocoBlock/suggestions#2579]
Now Jetforms allows anonymous users to upload files (great!), but in the same way that there is a button to choose file to upload from the CPT / CCT wordpress administration panel, could you put one to download the file?

![Captura](https://user-images.githubusercontent.com/57798462/106437172-48c8c300-6475-11eb-9672-3eb832abd345.JPG)

**Comment by carlosmungz** [comment]
I support your request, we need this functionality, I can't find it anywhere

**Comment by rh-hridoy12** [comment]
yes we need download file function as well. download all button to download all the files that have same category.
